src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/viewmodels/IndexViewModel.kt

```diff
@@ -45,11 +46,21 @@ data class IndexViewModel(@Serialise("reportsJson") val reports: List<ReportRowV
             var currentKey = 0
             val reportRows = reports.groupBy { it.name }.flatMap {
                 currentKey += 1
-                val parent = ReportRowViewModel.buildParent(currentKey, it.value, emptyCustomFields)
+
+                val parentTags = if (reportTags.containsKey(it.key))
+                {
+                    reportTags[it.key]!!
+                }
+                else
+                {
+                    listOf()
+                }
+
+                val parent = ReportRowViewModel.buildParent(currentKey, it.value, emptyCustomFields, parentTags)
 
                 val children = it.value.sortedByDescending { v -> v.date }.map { version ->
                     currentKey += 1
-                    ReportRowViewModel.buildVersion(version, currentKey, parent)
+                    ReportRowViewModel.buildVersion(version, currentKey, parent, version.tags)
```

Comment:
This method gets the whole `version` object as an argument so seems a bit weird to split out `version.tags` as an explicit extra argument?